sqlglot/parser.py

```diff
@@ -254,6 +254,11 @@ def _parse_drop(self):
         return exp.Drop(this=this, exists=exists, kind=kind)
 
     def _parse_create(self):
+        if self._match(TokenType.TEMPORARY):
```

Comment:
temporary = self._match(TokenType.TEMPORARY)

Reply:
Fixed had to wrap with bool as match can return a token.